package.json

```diff
@@ -4,13 +4,15 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "nest build",
+    "build": "yarn generate-abis && nest build",
     "format": "prettier --write .",
     "format-check": "prettier --check .",
+    "generate-abis": "node ./scripts/generate-abis.js",
+    "postinstall": "yarn generate-abis",
```

Comment:
Is this still needed (`generate-abis` is already called on the build step)?

Reply:
I think so - if you clone the project, you need the ABIs to run it. I considered adding it to the `start` scripts too but thought it was too much.